_conf/solr_entrypoint

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+# This script uses the following environment variables to control how SOLR is started and configured
+# SOLR_MODE cloud|standalone
+# SOLR_MEMORY <solr-memory-allocation-spec>
+
+SOLR_MODE=${SOLR_MODE:-cloud}
```

I would suggest not to use default arguments, because its difficult to track errors of misconfiguration, (e.g. if SOLR_MODE is wrongly spelled in docker-compose.yml). Instead its better to error out if the required configurations are missing/unspecified.